### ISSUE TITLE
[Issue]: Some LORA files from Civitai throw assertion errors. The same files works on vanilla A1111.

### ISSUE THREAD

**Issue by DrMeo** [vladmandic/automatic#764]
### Issue Description

When using several LORA files from civitai, Vlad's automatic throws assertion errors causing the LORA to not be applied to the image. 

Vanilla A1111 will use the same LORA file normally, on any model, and the LORA gets applied to the image as intended.

The assertion errors are: 
```
Traceback (most recent call last) ──────────────────────────────────────────╮
│ L:\vladsA1111\extensions-builtin\a1111-sd-webui-lycoris\..\..\extensions-builtin/Lora\lora.py:215 in load_loras      │
│                                                                                                                      │
│   214 │   │   │   │   try:                                                                                           │
│ ❱ 215 │   │   │   │   │   lora = load_lora(name, lora_on_disk.filename)                                              │
│   216 │   │   │   │   except Exception as e:                                                                         │
│                                                                                                                      │
│ L:\vladsA1111\extensions-builtin\a1111-sd-webui-lycoris\..\..\extensions-builtin/Lora\lora.py:185 in load_lora       │
│                                                                                                                      │
│   184 │   │   else:                                                                                                  │
│ ❱ 185 │   │   │   assert False, f'Bad Lora layer name: {key_diffusers} - must end in lora_up.w                       │
│   186                                                                                                                │
╰──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
AssertionError: Bad Lora layer name: lora_unet_down_blocks_0_downsamplers_0_conv.lora_mid.weight - must end in
lora_up.weight, lora_down.weight or alpha
```

The LORA That is misbehaving on this fork is https://civitai.com/models/21728?modelVersionId=37932 
The full models on that page are working, but the LORA/LOCON only works in Automatic1111. 

While searching for any info on the console error I found another report on civitai that this fork was not working on another LORA, while A1111 still was. This LORA can be found here https://civitai.com/models/26192

### Version Platform Description

Version: c470f399 Thu May 4 16:55:41 2023 -0400

Windows 11
Chrome browser
Issue also present on version 5dcaaba6

**Comment by vladmandic** [comment]
I'll check.
Any extensions you have installed either in A1111 or this repo?

**Comment by DrMeo** [comment]
Both vanilla and this repo have TagComplete and OpenOutpaint installed, and vanilla A1111 also has Dreambooth. Other than that just the default / built in extensions as far as I can remember. 

The vanilla install I have is an older commit though, I haven't updated it to torch2 yet. I plan on cloning the newest commit tomorrow to test if the torch2 ( and everything else being newer) change is what is affecting the LORA loading somehow.

**Comment by vladmandic** [comment]
Those are not vanilla Lora's, they are LoCONs (and it actually says that in Civit AI page)

And no chance that vanilla A1111 works: see [code](https://github.com/AUTOMATIC1111/stable-diffusion-webui/blob/5ab7f213bec2f816f9c5644becb32eb72c8ffb89/extensions-builtin/Lora/lora.py#L171)

So most likely you do have an extension that patches A1111 built-in Lora support

(Actually, if you *ignore* the error, rendering with lora still works, but it just skips that part of Lora)

I actually do have full support for LoCON and LyCORIS, but instead of patching LORA (because its not really a LORA), it works in a different namespace - use `<lyco:name>` instad of `<lora:name>`

I've changed the error messaging so it doesn't print silly error message and now it prints this:

```log
Loading weights: /home/vlado/dev/automatic/models/Lora/aiomonstergirls_aiov2LoraLocon.safetensors ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 0.0/213.0 MB -:--:--
10:02:55-663608 WARNING  Unknown Lora layer: lora_unet_down_blocks_0_downsamplers_0_conv.lora_mid.weight
10:02:55-665069 WARNING  Try using LyCORIS instead
Progress 4.12it/s ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 100% 0:00:00 0:00:02
```

**Comment by DrMeo** [comment]
Huh, interesting. 

You are right about the locon, it works with putting in the lycoris folder and calling it. 

I finally tracked it down to the confusion coming from in A1111 you had to put it in the models/LORA folder and call it as a LORA if you had the A1111-locon extension installed. I finally noticed the locon extension wasn't a base install extension ( or isn't anymore if it was ) after installing the latest A1111 and it not working there until that extension was installed.

**Comment by vladmandic** [comment]
yup. and locon extension is basically an older version of lycoris extension i have as built-in (same author). actual locon extension is eol and superseeded by lycoris as there were cases where patching <lora:name> namespace caused other problems, its far better to have a new extra network namespace <lyco:name>

**Comment by ChiaYen-Kan** [comment]
I have same error, at version a7755ac6d82416925de90bbc3b33ef3d6342e0de
i try put model into Lora or LyCORIS folder, but get same error
how fix it ?

```
Using VENV: C:\AI\automatic-new\venv
11:34:05-966786 INFO     Starting SD.Next
11:34:05-970787 INFO     Python 3.10.11 on Windows
11:34:06-073810 INFO     Version: app=sd.next updated=2023-09-13 hash=a7755ac6
                         url=https://github.com/vladmandic/automatic.git/tree/master
11:34:06-624659 INFO     Latest published version: d4871414ea5cd2bbb87532cedea04b7d9e4d5b94 2023-09-13T21:26:15Z
11:34:06-633926 INFO     Platform: arch=AMD64 cpu=AMD64 Family 25 Model 80 Stepping 0, AuthenticAMD system=Windows
                         release=Windows-10-10.0.19045-SP0 python=3.10.11
11:34:06-635926 DEBUG    Setting environment tuning
11:34:06-637926 DEBUG    Torch overrides: cuda=False rocm=False ipex=False diml=True openvino=False
11:34:06-639927 DEBUG    Torch allowed: cuda=False rocm=False ipex=False diml=True openvino=False
11:34:06-640927 INFO     Using DirectML Backend
11:34:06-777958 DEBUG    Repository update time: Wed Sep 13 21:16:44 2023
11:34:06-779958 DEBUG    Previous setup time: Thu Sep 14 10:23:51 2023
11:34:06-781958 INFO     Extensions: disabled=[]
11:34:06-782959 INFO     Extensions: enabled=['a1111-sd-webui-lycoris', 'clip-interrogator-ext', 'LDSR', 'Lora',
                         'multidiffusion-upscaler-for-automatic1111', 'ScuNET', 'sd-extension-system-info',
                         'sd-webui-agent-scheduler', 'sd-webui-controlnet', 'stable-diffusion-webui-images-browser',
                         'stable-diffusion-webui-rembg', 'SwinIR'] extensions-builtin
11:34:06-794961 INFO     Extensions: enabled=['Stable-Diffusion-Webui-Civitai-Helper'] extensions
11:34:06-797962 DEBUG    Latest extensions time: Thu Sep 14 10:23:33 2023
11:34:06-798962 DEBUG    Timestamps: version:1694611004 setup:1694658231 extension:1694658213
11:34:06-800962 INFO     No changes detected: Quick launch active
11:34:06-801963 INFO     Verifying requirements
11:34:06-823968 INFO     Verifying packages
11:34:06-825968 INFO     Extensions: disabled=[]
11:34:06-827969 INFO     Extensions: enabled=['a1111-sd-webui-lycoris', 'clip-interrogator-ext', 'LDSR', 'Lora',
                         'multidiffusion-upscaler-for-automatic1111', 'ScuNET', 'sd-extension-system-info',
                         'sd-webui-agent-scheduler', 'sd-webui-controlnet', 'stable-diffusion-webui-images-browser',
                         'stable-diffusion-webui-rembg', 'SwinIR'] extensions-builtin
11:34:06-839972 INFO     Extensions: enabled=['Stable-Diffusion-Webui-Civitai-Helper'] extensions
11:34:06-850991 INFO     Extension preload: {'extensions-builtin': 0.01, 'extensions': 0.0}
11:34:06-852985 DEBUG    Starting module: <module 'webui' from 'C:\\AI\\automatic-new\\webui.py'>
11:34:06-854982 INFO     Command line args: ['--use-directml', '--models-dir', 'C:/AI/Models', '--medvram', '--debug']
                         medvram=True models_dir=C:/AI/Models debug=True use_directml=True
11:34:14-328238 DEBUG    Loaded packages: torch=2.0.0+cpu diffusers=0.20.2 gradio=3.43.2
11:34:14-839354 DEBUG    Reading: config.json len=34
11:34:14-885364 INFO     Engine: backend=Backend.ORIGINAL compute=directml mode=no_grad device=privateuseone:0
11:34:14-888147 INFO     Device:
11:34:15-576522 DEBUG    Entering start sequence
11:34:15-576522 INFO     Using models path: C:/AI/Models
11:34:15-576522 DEBUG    Initializing
11:34:15-576522 INFO     Available VAEs: C:/AI/Models\VAE items=4
11:34:15-576522 DEBUG    Reading: cache.json len=2
11:34:15-592150 DEBUG    Reading: metadata.json len=18
11:34:15-598656 INFO     Available models: C:/AI/Models\Stable-diffusion items=6 time=0.02s
11:34:15-629920 DEBUG    Loading extensions
11:34:19-405817 INFO     Extension: script='extensions-builtin\sd-webui-controlnet\scripts\controlnet.py' Warning:
                         ControlNet failed to load SGM - will use LDM instead.
11:34:19-410158 INFO     Extension: script='extensions-builtin\sd-webui-controlnet\scripts\controlnet.py' ControlNet
                         preprocessor location:
                         C:\AI\automatic-new\extensions-builtin\sd-webui-controlnet\annotator\downloads
11:34:19-420160 INFO     Extension: script='extensions-builtin\sd-webui-controlnet\scripts\hook.py' Warning: ControlNet
                         failed to load SGM - will use LDM instead.
11:34:20-364372 INFO     Extension: script='extensions\Stable-Diffusion-Webui-Civitai-Helper\scripts\civitai_helper.py'
                         Civitai Helper: Get Custom Model Folder
11:34:20-367374 INFO     Extension: script='extensions\Stable-Diffusion-Webui-Civitai-Helper\scripts\civitai_helper.py'
                         Civitai Helper: Load setting from:
                         extensions\Stable-Diffusion-Webui-Civitai-Helper\setting.json
11:34:20-370095 INFO     Extension: script='extensions\Stable-Diffusion-Webui-Civitai-Helper\scripts\civitai_helper.py'
                         Civitai Helper: No setting file, use default
11:34:20-428111 INFO     Extensions time: 4.78s { a1111-sd-webui-lycoris=2.37s Lora=0.18s
                         sd-webui-agent-scheduler=0.39s sd-webui-controlnet=0.70s
                         stable-diffusion-webui-images-browser=0.16s stable-diffusion-webui-rembg=0.67s SwinIR=0.06s
                         Stable-Diffusion-Webui-Civitai-Helper=0.06s ScuNET=0.06s }
11:34:20-439114 DEBUG    Loaded upscalers: items=10
11:34:20-486819 INFO     Loading UI theme: name=black-teal style=Auto
11:34:20-497821 DEBUG    Loaded styles: folder=C:/AI/Models\styles items=38
11:34:20-501822 DEBUG    Creating UI
11:34:20-520827 DEBUG    Reading: ui-config.json len=0
11:34:20-565837 DEBUG    Extra networks: page='checkpoints' items=6 subdirs=1 tab=txt2img
                         dirs=['C:/AI/Models\\Stable-diffusion', 'C:/AI/Models\\Diffusers',
                         'C:\\AI\\Models\\Stable-diffusion'] time=0.01
11:34:20-582841 DEBUG    Extra networks: page='styles' items=38 subdirs=1 tab=txt2img dirs=['C:/AI/Models\\styles']
                         time=0.01
11:34:20-592844 DEBUG    Extra networks: page='textual inversion' items=3 subdirs=1 tab=txt2img
                         dirs=['C:/AI/Models/embeddings'] time=0.01
11:34:20-601845 DEBUG    Extra networks: page='hypernetworks' items=0 subdirs=0 tab=txt2img
                         dirs=['C:/AI/Models\\hypernetworks'] time=0.0
11:34:20-621850 DEBUG    Extra networks: page='lora' items=11 subdirs=1 tab=txt2img dirs=['C:/AI/Models\\Lora']
                         time=0.02
11:34:20-628851 DEBUG    Extra networks: page='lycoris' items=0 subdirs=0 tab=txt2img dirs=['C:/AI/Models\\LyCORIS']
                         time=0.0
11:34:20-875906 DEBUG    Reading: ui-config.json len=0
11:34:20-905375 INFO     Themes: builtin=6 default=5 external=48
11:34:21-760045 DEBUG    Script: 0.78s ui_tabs
                         C:\AI\automatic-new\extensions-builtin\stable-diffusion-webui-images-browser\scripts\image_bro
                         wser.py
11:34:22-631241 DEBUG    Extension list refresh: processed=221 installed=13 enabled=13 disabled=0 visible=221 hidden=0
11:34:22-927099 INFO     Local URL: http://127.0.0.1:7860/
11:34:22-929100 DEBUG    Gradio registered functions: 2095
11:34:22-930099 INFO     Initializing middleware
11:34:22-934100 DEBUG    Creating API
11:34:23-105443 INFO     [AgentScheduler] Task queue is empty
11:34:23-107646 INFO     [AgentScheduler] Registering APIs
11:34:23-230664 DEBUG    Scripts setup: ['Tiled Diffusion:0.028s', 'ControlNet:0.059s', 'X/Y/Z grid:0.005s']
11:34:23-233665 DEBUG    Model metadata: metadata.json no changes
11:34:23-448805 DEBUG    Select checkpoint: model lastpiececore_a0751+hassakuHentaiModel_v13.safetensors [00ff6f10b1]
11:34:23-450805 DEBUG    Load model weights: existing=False
                         target=C:\AI\Models\Stable-diffusion\lastpiececore_a0751+hassakuHentaiModel_v13.safetensors
                         info=None
Loading weights: C:\AI\Models\Stable-diffusion\lastpiececore_a0751+hassakuHentaiModel_v13.safetensors ----- 0.0… -:--: Loading weights: C:\AI\Models\Stable-diffusion\lastpiececore_a0751+hassakuHentaiModel_v13.safetensors ----- 0.0… -:--: Loading weights: C:\AI\Models\Stable-diffusion\lastpiececore_a0751+hassakuHentaiModel_v13.safetensors ----- 0.0… -:--: …
                                                                                                            GB
11:34:23-626704 DEBUG    Load model:
                         name=C:\AI\Models\Stable-diffusion\lastpiececore_a0751+hassakuHentaiModel_v13.safetensors
                         dict=True
11:34:23-628704 DEBUG    Verifying Torch settings
11:34:23-635706 INFO     Torch override dtype: no-half set
11:34:23-636706 INFO     Torch override VAE dtype: no-half set
11:34:23-637707 DEBUG    Desired Torch parameters: dtype=FP16 no-half=True no-half-vae=True upscast=True
11:34:23-639706 INFO     Setting Torch parameters: dtype=torch.float32 vae=torch.float32 unet=torch.float32
                         context=no_grad fp16=True bf16=False
11:34:23-642707 DEBUG    Torch default device: privateuseone:0
11:34:23-643708 DEBUG    Model dict loaded: {'ram': {'used': 0.85, 'total': 31.87}, 'gpu': {'used': 0.01, 'total':
                         0.01}, 'retries': 'DirectMLDevice', 'oom': 0}
11:34:23-657711 DEBUG    Model config loaded: {'ram': {'used': 0.85, 'total': 31.87}, 'gpu': {'used': 0.01, 'total':
                         0.01}, 'retries': 'DirectMLDevice', 'oom': 0}
11:34:24-034796 INFO     LDM: LatentDiffusion: Running in eps-prediction mode
11:34:24-036796 INFO     LDM: DiffusionWrapper has 859.52 M params.
11:34:24-037796 DEBUG    Model created from config: C:\AI\automatic-new\configs\v1-inference.yaml
11:34:24-039797 DEBUG    Model autodetect model:
                         C:\AI\Models\Stable-diffusion\lastpiececore_a0751+hassakuHentaiModel_v13.safetensors
                         pipeline=Stable Diffusion size=1.99 GB
11:34:24-041797 DEBUG    Model weights loading: {'ram': {'used': 1.82, 'total': 31.87}, 'gpu': {'used': 0.01, 'total':
                         0.01}, 'retries': 'DirectMLDevice', 'oom': 0}
Loading weights: C:/AI/Models\VAE\vae-ft-mse-840000-ema-pruned.safetensors ----------------------- 0.0/334.6 MB -:--:--
11:34:24-683942 DEBUG    Model weights loaded: {'ram': {'used': 7.15, 'total': 31.87}, 'gpu': {'used': 0.01, 'total':
                         0.01}, 'retries': 'DirectMLDevice', 'oom': 0}
11:34:24-688944 DEBUG    Model weights moved: {'ram': {'used': 7.15, 'total': 31.87}, 'gpu': {'used': 0.01, 'total':
                         0.01}, 'retries': 'DirectMLDevice', 'oom': 0}
11:34:24-697945 INFO     Applying sub-quadratic cross attention optimization
11:34:24-707947 INFO     Loaded embeddings: loaded=3 skipped=0
11:34:24-714949 INFO     Model loaded in 1.26s { load=0.18s create=0.38s apply=0.49s vae=0.15s }
11:34:25-066189 DEBUG    gc: collected=10440 device=privateuseone:0 {'ram': {'used': 7.15, 'total': 31.87}, 'gpu':
                         {'used': 0.01, 'total': 0.01}, 'retries': 'DirectMLDevice', 'oom': 0}
11:34:25-070189 INFO     Model load finished: {'ram': {'used': 7.15, 'total': 31.87}, 'gpu': {'used': 0.01, 'total':
                         0.01}, 'retries': 'DirectMLDevice', 'oom': 0} cached=0
11:34:25-176214 DEBUG    Saving: config.json len=1432
11:34:25-179550 INFO     Startup time: 18.29s { torch=5.91s gradio=0.66s diffusers=0.87s libraries=1.25s
                         extensions=4.78s ui-txt2img=0.21s ui-img2img=0.11s ui-settings=0.09s ui-extensions=1.70s
                         ui-defaults=0.07s launch=0.21s api=0.07s app-started=0.23s checkpoint=1.95s }
11:34:35-486858 INFO     Themes: builtin=6 default=5 external=48
11:34:46-849168 DEBUG    Paste prompt: nsfw, 1boy, (kawaii:1.2), cute, high resolution, (incredibly absurdres:1.1),
                         (hires.fix:1.3), anime visual, extremely detailed CG unity 8k wallpaper, (beautiful detailed
                         nude:1.5), (beautiful detailed flat chest:1.05), (detailed beautiful face:1.4), (futanari,
                         flaccid penis), (detailed beautiful eyes), (beautiful medium-length side ponytail:1.5), wavy
                         hair, braided bangs, blonde hair, (standing:1.5), light smile, bathroom, detailed background,
                         <lora:GoodHands-beta2:1.0>, <lora:futanari_concept_full_v1-000020:1.0>
                         Negative prompt: (easynegative), (negative_hand-neg:1.5), (badhandv4), (worst quality, low
                         quality:1.4), (bad hand, fused fingers, mutated hands, missing fingers, cloned), ((((pubic
                         hair, pussy, vangina, vaginal, multiple penises))))
                         Steps: 30, Seed: 3701548916, Sampler: DPM++ 2M SDE Karras, CFG scale: 7, Size: 512x768,
                         Parser: Full parser, Model: lastpiececore_a0751+hassakuHentaiModel_v13, Model hash:
                         00ff6f10b1, VAE: vae-ft-mse-840000-ema-pruned, Clip skip: 2, Prompt2: "high resolution,
                         (incredibly absurdres:1.1), (hires.fix:1.3), anime visual, extremely detailed CG unity 8k
                         wallpaper", Negative2: "(easynegative), (negative_hand-neg:1.5), (badhandv4), (bad hand, fused
                         fingers, mutated hands, missing fingers, cloned), (logo:2.0), (watermark:2.0)", Backend:
                         Original, Version: a7755ac, Comment: , Operations: none, Lora hashes: "GoodHands-beta2:
                         c05ed27929"
11:34:58-642778 DEBUG    txt2img: id_task=task(076yoih834qh69f)|prompt=nsfw, 1boy, (kawaii:1.2), cute, high resolution,
                         (incredibly absurdres:1.1), (hires.fix:1.3), anime visual, extremely detailed CG unity 8k
                         wallpaper, (beautiful detailed nude:1.5), (beautiful detailed flat chest:1.05), (detailed
                         beautiful face:1.4), (futanari, flaccid penis), (detailed beautiful eyes), (beautiful
                         medium-length side ponytail:1.5), wavy hair, braided bangs, blonde hair, (standing:1.5), light
                         smile, bathroom, detailed background, <lora:GoodHands-beta2:1.0>,
                         <lora:futanari_concept_full_v1-000020:1.0>|negative_prompt=(easynegative),
                         (negative_hand-neg:1.5), (badhandv4), (worst quality, low quality:1.4), (bad hand, fused
                         fingers, mutated hands, missing fingers, cloned), ((((pubic hair, pussy, vangina, vaginal,
                         multiple
                         penises))))|prompt_styles=[]|steps=30|sampler_index=6|latent_index=None|full_quality=True|rest
                         ore_faces=False|tiling=False|n_iter=1|batch_size=1|cfg_scale=7|clip_skip=2|seed=3701548916.0|s
                         ubseed=-1.0|subseed_strength=0|seed_resize_from_h=0|seed_resize_from_w=0||height=768|width=512
                         |enable_hr=False|denoising_strength=0.5|hr_scale=2|hr_upscaler=None|hr_force=False|hr_second_p
                         ass_steps=20|hr_resize_x=0|hr_resize_y=0|image_cfg_scale=6|diffusers_guidance_rescale=0.7|refi
                         ner_steps=5|refiner_start=0.8|refiner_prompt=high resolution, (incredibly absurdres:1.1),
                         (hires.fix:1.3), anime visual, extremely detailed CG unity 8k
                         wallpaper|refiner_negative=(easynegative), (negative_hand-neg:1.5), (badhandv4), (bad hand,
                         fused fingers, mutated hands, missing fingers, cloned), (logo:2.0),
                         (watermark:2.0)|override_settings_texts=[]
11:34:58-672872 INFO     Loaded VAE-approx model: C:\AI\automatic-new\models\VAE-approx\model.pt
Loading weights: C:\AI\Models\Lora\GoodHands-beta2.safetensors ----------------------------------- 0.0/540.6 MB -:--:--
Loading weights: C:\AI\Models\Lora\futanari_concept_full_v1-000020.safetensors ------------------- 0.0/704.7 MB -:--:--
11:34:59-780596 ERROR    loading Lora C:\AI\Models\Lora\futanari_concept_full_v1-000020.safetensors: AssertionError
┌───────────────────────────────────────── Traceback (most recent call last) ─────────────────────────────────────────┐
│ C:\AI\automatic-new\extensions-builtin\Lora\lora.py:253 in load_loras                                               │
│                                                                                                                     │
│   252 │   │   │   │   │   else:                                                                                     │
│ > 253 │   │   │   │   │   │   lora = load_lora(name, lora_on_disk)                                                  │
│   254 │   │   │   │   except Exception as e:                                                                        │
│                                                                                                                     │
│ C:\AI\automatic-new\extensions-builtin\Lora\lora.py:217 in load_lora                                                │
│                                                                                                                     │
│   216 │   │   else:                                                                                                 │
│ > 217 │   │   │   raise AssertionError(f"Bad Lora layer name: {key_diffusers} - must end in lo                      │
│   218                                                                                                               │
└─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
AssertionError: Bad Lora layer name: lora_te_text_model_encoder_layers_0_mlp_fc1.diff - must end in lora_up.weight,
lora_down.weight or alpha
11:35:00-553781 DEBUG    Sampler: sampler=DPM++ 2M SDE Karras config={'scheduler': 'karras', 'brownian_noise': True,
                         'discard_next_to_last_sigma': True}
 23%|███████████████████▎                                                               | 7/30 [00:57<02:59,  7.78s/it]11:35:59-863107 DEBUG    Server alive=True jobs=2 requests=76 uptime=105s memory used=12.43 total=31.87 job="txt2img"
                         0/1
 73%|████████████████████████████████████████████████████████████▏                     | 22/30 [02:49<01:00,  7.50s/it]11:37:51-054668 DEBUG    Requested interrupt
 77%|██████████████████████████████████████████████████████████████▊                   | 23/30 [02:56<00:52,  7.50s/it]11:37:59-554712 DEBUG    Server alive=True jobs=3 requests=157 uptime=225s memory used=20.54 total=31.87 idle
 90%|█████████████████████████████████████████████████████████████████████████▊        | 27/30 [03:26<00:22,  7.49s/it]11:38:33-104473 DEBUG    Requested interrupt
11:38:34-101712 DEBUG    Requested interrupt
 93%|████████████████████████████████████████████████████████████████████████████▌     | 28/30 [03:34<00:15,  7.66s/it]
11:38:41-101117 DEBUG    Saving: image=C:/AI/outputs/text\2023-09-14\00010-20230914113841.png type=PNG size=512x768
11:38:41-245149 INFO     Processed: images=1 time=222.58s its=0.13 memory={'ram': {'used': 23.76, 'total': 31.87},
                         'gpu': {'used': 15.86, 'total': 30.38}, 'retries': 'DirectMLDevice', 'oom': 0}
11:38:46-369465 INFO     Deleting image: C:\AI\outputs\text\2023-09-14\00010-20230914113841.png
11:38:51-637129 INFO     Available models: C:/AI/Models\Stable-diffusion items=6 time=0.00s
11:38:51-645131 DEBUG    Extra networks: page='checkpoints' items=6 subdirs=1 tab=txt2img
                         dirs=['C:/AI/Models\\Stable-diffusion', 'C:/AI/Models\\Diffusers',
                         'C:\\AI\\Models\\Stable-diffusion'] time=0.01
11:38:51-651131 DEBUG    Refreshing Extra networks: page='Checkpoints' items=6 tab=txt2img
11:39:00-132803 DEBUG    Extra networks: page='lycoris' items=1 subdirs=1 tab=txt2img dirs=['C:/AI/Models\\LyCORIS']
                         time=0.0
11:39:00-136804 DEBUG    Refreshing Extra networks: page='LyCORIS' items=1 tab=txt2img
11:39:10-828232 DEBUG    txt2img: id_task=task(0nh7ti21skxq3fk)|prompt=nsfw, 1boy, (kawaii:1.2), cute, high resolution,
                         (incredibly absurdres:1.1), (hires.fix:1.3), anime visual, extremely detailed CG unity 8k
                         wallpaper, (beautiful detailed nude:1.5), (beautiful detailed flat chest:1.05), (detailed
                         beautiful face:1.4), (futanari, flaccid penis), (detailed beautiful eyes), (beautiful
                         medium-length side ponytail:1.5), wavy hair, braided bangs, blonde hair, (standing:1.5), light
                         smile, bathroom, detailed background, <lora:GoodHands-beta2:1.0>,
                         <lyco:futanari_concept_full_v1-000020:1.0>|negative_prompt=(easynegative),
                         (negative_hand-neg:1.5), (badhandv4), (worst quality, low quality:1.4), (bad hand, fused
                         fingers, mutated hands, missing fingers, cloned), ((((pubic hair, pussy, vangina, vaginal,
                         multiple
                         penises))))|prompt_styles=[]|steps=30|sampler_index=6|latent_index=None|full_quality=True|rest
                         ore_faces=False|tiling=False|n_iter=1|batch_size=1|cfg_scale=7|clip_skip=2|seed=3701548916.0|s
                         ubseed=-1.0|subseed_strength=0|seed_resize_from_h=0|seed_resize_from_w=0||height=768|width=512
                         |enable_hr=False|denoising_strength=0.5|hr_scale=2|hr_upscaler=None|hr_force=False|hr_second_p
                         ass_steps=20|hr_resize_x=0|hr_resize_y=0|image_cfg_scale=6|diffusers_guidance_rescale=0.7|refi
                         ner_steps=5|refiner_start=0.8|refiner_prompt=high resolution, (incredibly absurdres:1.1),
                         (hires.fix:1.3), anime visual, extremely detailed CG unity 8k
                         wallpaper|refiner_negative=(easynegative), (negative_hand-neg:1.5), (badhandv4), (bad hand,
                         fused fingers, mutated hands, missing fingers, cloned), (logo:2.0),
                         (watermark:2.0)|override_settings_texts=[]
Loading weights: C:/AI/Models\LyCORIS\futanari_concept_full_v1-000020.safetensors ---------------- 0.0/704.7 MB -:--:--
11:39:10-898988 ERROR    activating extra network lyco with arguments [<modules.extra_networks.ExtraNetworkParams
                         object at 0x0000023F89FCD960>]: AssertionError
┌───────────────────────────────────────── Traceback (most recent call last) ─────────────────────────────────────────┐
│ C:\AI\automatic-new\modules\extra_networks.py:71 in activate                                                        │
│                                                                                                                     │
│    70 │   │   try:                                                                                                  │
│ >  71 │   │   │   extra_network.activate(p, extra_network_args)                                                     │
│    72 │   │   except Exception as e:                                                                                │
│                                                                                                                     │
│ C:\AI\automatic-new\extensions-builtin\a1111-sd-webui-lycoris\extra_networks_lyco.py:90 in activate                 │
│                                                                                                                     │
│   89 │   │   │   self.cache = all_lycos                                                                             │
│ > 90 │   │   lycoris.load_lycos(names, te_multipliers, unet_multipliers, dyn_dims)                                  │
│   91                                                                                                                │
│                                                                                                                     │
│ C:\AI\automatic-new\extensions-builtin\a1111-sd-webui-lycoris\lycoris.py:488 in load_lycos                          │
│                                                                                                                     │
│   487 │   │   │   if lyco is None or os.path.getmtime(lyco_on_disk.filename) > lyco.mtime:                          │
│ > 488 │   │   │   │   lyco = load_lyco(name, lyco_on_disk.filename)                                                 │
│   489                                                                                                               │
│                                                                                                                     │
│ C:\AI\automatic-new\extensions-builtin\a1111-sd-webui-lycoris\lycoris.py:458 in load_lyco                           │
│                                                                                                                     │
│   457 │   │   else:                                                                                                 │
│ > 458 │   │   │   assert False, f'Bad Lyco layer name: {key_diffusers} - must end in lyco_up.w                      │
│   459                                                                                                               │
└─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
AssertionError: Bad Lyco layer name: lora_te_text_model_encoder_layers_0_mlp_fc1.diff_b - must end in lyco_up.weight,
lyco_down.weight or alpha
11:39:11-299004 DEBUG    Sampler: sampler=DPM++ 2M SDE Karras config={'scheduler': 'karras', 'brownian_noise': True,
                         'discard_next_to_last_sigma': True}
 10%|████████▎                                                                          | 3/30 [00:23<03:28,  7.72s/it]11:39:40-797837 DEBUG    Requested interrupt
 13%|███████████                                                                        | 4/30 [00:30<03:21,  7.74s/it]
11:39:48-286549 DEBUG    Saving: image=C:/AI/outputs/text\2023-09-14\00010-20230914113948.png type=PNG size=512x768
11:39:48-449586 INFO     Processed: images=1 time=37.60s its=0.80 memory={'ram': {'used': 23.79, 'total': 31.87},
                         'gpu': {'used': 15.86, 'total': 30.38}, 'retries': 'DirectMLDevice', 'oom': 0}
11:40:00-330620 DEBUG    Server alive=True jobs=5 requests=209 uptime=346s memory used=23.79 total=31.87 idle
```

**Comment by vladmandic** [comment]
Valid issue, but given it doest work even with lyco, this is not the same as original issue. Plus original is closed 4 months ago - can you please create a new issue so this doesn't fall off the radar?
